### PR TITLE
Update DeleteCommand, Task and UI

### LINKED ISSUE #19: Create TaskRestore class
Class is to provide a string to print out when the delete command is performed.
String is the user input to remake the task

### PR DESCRIPTION
**Pull request by Kafcis**
Added the restoration command into the task objects
Changed the DeleteCommand and UI to take in the additional arguments
The main Athena class has some code to verify if the changes were functional
Checkstyle not accounted for yet

### CODE REVIEW

File: src/main/java/athena/DeleteCommand.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 public class DeleteCommand extends Command {
     private int deleteIndex;
 
+
```
Comment (amanda-chua) [inline src/main/java/athena/DeleteCommand.java]: Remove extra line

File: src/main/java/athena/Ui.java [hunk 2]
```diff
@@ -63,8 +63,10 @@ public void printTaskEdited(Task task) {
      * Prints out a message verifying that the task the user specified has been deleted.
      * @param task The task that the user has deleted.
      */
-    public void printTaskDeleted(Task task) {
+    public void printTaskDeleted(Task task,String taskRestore) {
```
Comment (amanda-chua) [inline src/main/java/athena/Ui.java:93]: add space before String

**Review by amanda-chua — APPROVED** [review]
Nice

File: src/main/java/athena/Athena.java [hunk 3]
```diff
@@ -27,11 +27,14 @@ public static void main(String[] args) {
...
-        command = parser.parse("help");
-        command.execute(taskList, athena.ui);
+//        command = parser.parse("help");
+//        command.execute(taskList, athena.ui);
         command = parser.parse("list");
         command.execute(oldTask, athena.ui);
         storage.saveTaskListData(taskList);
+        command= parser.parse("delete 2");
+        command.execute(taskList, athena.ui);
+        storage.saveTaskListData(taskList);
```
Comment (daniellimws) [inline src/main/java/athena/Athena.java:39]: We should put these things in our JUnit tests. I think it's ok if you want to do it in a different pull request.

**Review by daniellimws — APPROVED** [review]
Nice
